scripts/update-commit-latest-env.py

```
@@ -0,0 +1,102 @@
+    tasks = [get_image_vcs_ref(image) for image in images_to_inspect]
+    return await asyncio.gather(*tasks)
+
+
+async def main():
+    with open(PROJECT_ROOT / "manifests/base/params-latest.env", "rt") as file:
+        images_to_inspect: list[list[str]] = [line.strip().split('=', 1) for line in file.readlines()
+                                              if line.strip() and not line.strip().startswith("#")]
+
+    results = await inspect(value for _, value in images_to_inspect)
+    output = []
+    for image, result in zip(images_to_inspect, results):
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)